qiskit_experiments/library/calibration/fine_frequency_cal.py

```diff
@@ -0,0 +1,138 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine frequency calibration experiment."""
+
+from typing import List, Optional
+import numpy as np
+
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import ExperimentData
+from qiskit_experiments.calibration_management.update_library import BaseUpdater
+from qiskit_experiments.calibration_management import (
+    BaseCalibrationExperiment,
+    Calibrations,
+)
+from qiskit_experiments.library.characterization.fine_frequency import FineFrequency
+
+
+class FineFrequencyCal(BaseCalibrationExperiment, FineFrequency):
+    """A calibration version of the fine frequency experiment.
+    
+    # section: see_also
+        :py:class:`FineFrequency`
+    """
+
+    def __init__(
+        self,
+        qubit: int,
+        calibrations: Calibrations,
+        backend: Optional[Backend] = None,
+        repetitions: List[int] = None,
+        auto_update: bool = True,
+    ):
+        r"""see class :class:`FineFrequency` for details.
+
+        Note that this class implicitly assumes that the target angle of the gate
+        is :math:`\pi` as seen from the default experiment options.
```

Comment:
Actually, this one is legit. Since the fine freq does `sx - delay - rz(n * pi/2) - sx` we have a pi/2 angle per gate. Perhaps the wording is a bit confusing since we are doing this with virtual Z rotations?

Reply:
Hmm, it should be `:math: \pi / 2` though, right? Also, the reference to a gate is a little funny here because we are calibrating the frequency. When this wording is used in other cals, "target angle" is referring to the amount of rotation produced by the gate being calibrated. Perhaps the comment should also clarify that the "gate" being calibrated is a finite duration `rz(pi / 2)` gate with any error attributed to a frequency offset?

Reply:
How about this? 737834c